db/migrate/20230222155444_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorId
+      t.integer :postId
```

Comment:
- [ ] Please 🙏🏽 use references it will create the corresponding columns for author_id and post_id and the corresponding index.
- **HINT**: 
- `t.references :author, foreign_key: { to_table: 'users' }`
- `t.references :post, foreign_key: true`